build/int.cloudbuild.yaml

```diff
@@ -487,5 +487,3 @@ tags:
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
   _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.23'
-options:
-  machineType: 'E2_HIGHCPU_32'
```

Comment:
Should we revert to the previous type `N1_HIGHCPU_8` or the e2 variant? Some of these tests run in parallel stages on same host so I think it would still benefit from higher cpu

Reply:
The `N1_HIGHCPU_8` is deprecated, but I can test against the replacement `E2-HIGHCPU-8`.  This (`E2-STANDARD-2`) was already within 10% of the runtime with the previous `E2_HIGHCPU_32`.

Reply:
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/2284